pages/index.js

```diff
@@ -16,56 +16,61 @@ export default function Home({userName, handleUserNameFormSubmit}) {
-            <Link href="https://pixabay.com/vectors/arm-exercise-fist-flex-health-2029406/">
-              <StyledParagraph>Muscle Arm</StyledParagraph>
-            </Link>
-            <StyledParagraph>by OpenClipart-Vectors.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/vectors/man-running-silhouette-black-40739/">
-              <StyledParagraph>Man Running Silhouette</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Clker-Free-Vector-Images.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/photos/action-jump-woman-exercise-figure-1850677/">
-              <StyledParagraph>Action Jump Woman Exercise.</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Pexels</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/vectors/search-find-magnifier-zoom-glass-6699087/">
-              <StyledParagraph>Magnifiyng Glass</StyledParagraph>
-            </Link>
-            <StyledParagraph>by thehalaldesign.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/vectors/bookmark-label-reading-simple-6751603/">
-              <StyledParagraph>Boomkark</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Hedgye.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/vectors/bookmark-label-reading-simple-6751594/">
-              <StyledParagraph>Bookmark Star</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Hedgye.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://pixabay.com/vectors/gear-wheel-gears-toothed-wheel-304395/">
-              <StyledParagraph>Gear</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Clker-Free-Vector-Images.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://fontawesome.com/">
-              <StyledParagraph>Fontawesome</StyledParagraph>
-            </Link>
-            <StyledParagraph>
-              Bookmark-regular, Bookmark-solid, bowl-food-solid, dumbbell-solid.
-            </StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-            <Link href="https://fonts.google.com/specimen/Roboto">
-              <StyledParagraph>Roboto Font</StyledParagraph>
-            </Link>
-            <StyledParagraph>by Google.</StyledParagraph>
-            <StyledParagraph>---</StyledParagraph>
-          </FormContainer>
+          {userName === "DontRender" ? (
+            ""
+          ) : (
+            <FormContainer>
+              <StyledParagraphNormal>Credits</StyledParagraphNormal>
+              <Link href="https://pixabay.com/vectors/arm-exercise-fist-flex-health-2029406/">
```

Comment:
maybe add some aria-labels  to the links

Reply:
I am not sure, but here it seems I dont need aria lables, because it is a link and inside the link is the description of where it leads to.